mindsdb/integrations/handlers/vertex_handler/__init__.py

```diff
@@ -0,0 +1,18 @@
+from mindsdb.integrations.libs.const import HANDLER_TYPE
+
+from .__about__ import __version__ as version, __description__ as description
+
+try:
+    from .vertex_handler import VertexHandler as Handler
+
+    import_error = None
+except Exception as e:
+    Handler = None
+    import_error = e
+
+title = "Vertex"
+name = "vertex"
+type = HANDLER_TYPE.ML
+permanent = True
```

Comment:
this handler required credentials, so it can't be permanent. it will be used via ml engine (next comment)

Reply:
Thanks - have dealt with this now

Reply:
So then we can remove "from files (select * from vertext_anomaly_detection)" from command
```
create model ...
from files 
  (select * from vertext_anomaly_detection)
predict ..
Using ...